pkg/app/src/components/OrganisationCard/item.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+
+import { fetchIpfsJson } from 'src/utils/ipfs'
+
+import { TileCard } from './modules/tileCard'
+
+export function Item({ data }) {
+	const [metadata, setMetadata] = useState<any>()
```

Comment:
Do not use any, unless absolutely necessary. 
there is a folder @types where you can create an interface for the metadata.